### PR TITLE
[v0.5.0] Update Kafka WS API documentation

### PR DESCRIPTION
**Pull request by eduardogmisiuk**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR?** (Link to an issue, e.g. #99999)
https://github.com/dojot/dojot/issues/1584

* **Other information**:

### CODE REVIEW

**Review by mprevide — DISMISSED** [review]

File: subscription-engine/kafka-ws/README.md [hunk 1]
```diff
@@ -316,7 +343,25 @@ If you are developing, you can use `nodemon` too:
 ```shell
 npm run dev
 ```
+__NOTE THAT_ in order to use WebSocket with nginx, Kong, API gateway or similar, look for timeout
```
Comment (thiagodpf) [inline subscription-engine/kafka-ws/README.md:346]: missing an underscore...

> `__NOTE THAT_`

> `__NOTE THAT__`

Comment (thiagodpf) [inline subscription-engine/kafka-ws/README.md:346]: I would put the **N**ginx software with the first letter capitalized just like **K**ong, but it is just a suggestion.

File: subscription-engine/kafka-ws/docs/apiary.apib [hunk 2]
```diff
@@ -6,15 +6,17 @@ FORMAT: 1A
 The Kafka WS service provides support for pure WebSocket connections to retrieve data directly
 from Kafka.
 
-# Group Ticket
+# Group ticket
```
Comment (thiagodpf) [inline subscription-engine/kafka-ws/docs/apiary.apib:9]: The word `Group` is actually a _marker_, it is not displayed, so why is the word **t**icket changed to lowercase while the next `Group` marker has the word **W**ebSocket in uppercase? Just my curiosity...

**Review by mprevide — APPROVED** [review]
I think the latest changes need to be merge into the development branch too

**Review by thiagodpf — APPROVED** [review]
:+1: :100:

**Pull request merged**